### ISSUE TITLE
Define a max number of verions to store

### ISSUE THREAD

**Issue by icarito** [publiclab/mapknitter#1167]
Quoting @jywarren : 
"come up with a rubric for max # of versions created PER PERIOD" (...) 
this would require some planning and policy discussion.

Not sure how this relates to the features but the thing is that paper-trail gem seems to be related to slowness and it suggests for performance to "limit the number of versions".

From a DBA point of view, currently our heaviest table is "warpables" with 268723 records.
Can we do something to reduce or optimize this?
Thanks!

### Relevant URLs

https://

### PublicLab.org username

(to help reproduce the issue)

### Browser, version, and operating system

For bug reports, fill out the above template; for feature requests, you can delete the template.

**Comment by icarito** [comment]
Actually, talking about heaviest, our "versions" table has 489978 records!

**Comment by icarito** [comment]
And our "nodes" table has 2746181 records, that is 2.7M records.

**Comment by jywarren** [comment]
oh no-- versions i think is a relatively new table! Could it be that it's
set to save at the wrong trigger, and is just saving endlessly?

**Comment by icarito** [comment]
I don't know but I believe this is slowing the gallery down. While deploying to the Google Cloud helped with performance, this route remains slow (2s-6s)

**Comment by jywarren** [comment]
Noting we can limit # of saved versions with this config:

https://github.com/paper-trail-gem/paper_trail#2e-limiting-the-number-of-versions-created

**Comment by jywarren** [comment]
Attempted this in #1312 !

**Comment by jywarren** [comment]
That is merged and ready for testing in stable! We can also monitor performance in relation afterwards.

**Comment by jywarren** [comment]
Confirmed on stable.mapknitter.org that only 20 versions may be saved!

https://stable.mapknitter.org/maps/unc-charlotte-balloon-mapping--2/edit

![image](https://user-images.githubusercontent.com/24359/82361233-916c4900-99d8-11ea-9ac4-eec86692c9d5.png)

**Comment by jywarren** [comment]
I don't think this removes past instances of too many versions though. Not sure!

**Comment by jywarren** [comment]
https://stable.mapknitter.org/maps/caldwell-campus/edit shows 38 revisions on an image, so it didn't trim retroactively:

![image](https://user-images.githubusercontent.com/24359/82361458-dc865c00-99d8-11ea-8a50-82e7384a1ef3.png)

**Comment by jywarren** [comment]
But i noticed that many images have only 2-5 revisions. So maybe it's not too bad... i had worried that thousands of versions were being saved...
